sdk/storage/azure-storage-queue/azure/storage/queue/_shared/policies.py

```diff
@@ -119,12 +118,6 @@ def urljoin(base_url, stub_url):
 class QueueMessagePolicy(SansIOHTTPPolicy):
 
     def on_request(self, request):
-        # Hack to fix generated code adding '/messages' after SAS parameters
-        includes_messages = request.http_request.url.endswith('/messages')
-        if includes_messages:
-            request.http_request.url = request.http_request.url[:-(len('/messages'))]
-            request.http_request.url = urljoin(request.http_request.url, 'messages')
-
```

Comment:
Oh, sorry I missed this the first time, but we REALLY need this specific to Queue :)